Resources/Prototypes/Recipes/Construction/utilities.yml

```diff
@@ -0,0 +1,80 @@
+#POWER
+- type: construction
+  name: APC
+  id: apc
+  graph: apc
+  startNode: start
+  targetNode: apc
+  category: Utilities
+  description: "Area Power Controller (APC). Controls power. In an area."
+  icon:
+    sprite: Structures/Power/apc.rsi
+    state: apc0
+  objectType: Structure
+  placementMode: SnapgridCenter
+  canBuildInImpassable: true
+
+- type: construction
+  name: Solar Assembly
+  id: SolarPanel
+  graph: solarpanel
+  startNode: start
+  targetNode: solarpanel
+  category: Utilities
+  description: "Can be turned into a solar panel or solar tracker."
+  icon:
+    sprite: Structures/Power/Generation/solar_panel.rsi
+    state: solar_assembly
+  objectType: Structure
+  placementMode: SnapgridCenter
+  canBuildInImpassable: false
+  conditions:
+    - !type:TileNotBlocked {}
+
+- type: construction
+  name: Cable Terminal
+  id: cableTerminal
+  graph: cable_terminal
+  startNode: start
+  targetNode: cable_terminal
+  category: Utilities
+  description: "Input of devices such as the SMES. The red cables needs to face the device."
+  icon:
+    sprite: Structures/Power/cable_terminal.rsi
+    state: term
+  objectType: Structure
+  placementMode: SnapgridCenter
+  canBuildInImpassable: false
```

Comment:
if these were moved from somewhere, it doesn't seem the originals were deleted

Reply:
Yeah I need the restructure to be merged first in my other PR. This was my attempt at a work around so that it passed tests but I didn't remove the original in this one. Either way it was going to fail tests.

As an aside I also messed up the directory for the graphs with a capital U and can't seem to change that so Idk, will wait for merges.